assembly/fabric8-ide-gwt-app/src/main/resources/org/eclipse/che/ide/public/IDE.html

```diff
@@ -0,0 +1,313 @@
+<!--
```

Comment:
why do we need the IDE.html from upstream che ?

Reply:
yes for now. See PR description

Reply:
I mean, I don't see why we need duplicates. IDE.html is in the upstream artifact so it can be copied as a dependency instead of adding it to source repository

Reply:
I tried to do it with maven-war-plugin
```
<plugin>
                <groupId>org.apache.maven.plugins</groupId>
                <artifactId>maven-war-plugin</artifactId>
                <configuration>
                    <overlays>
                        <overlay>
                            <groupId>org.eclipse.che</groupId>
                            <artifactId>che-ide-gwt-app</artifactId>
                            <type>war</type>
                            <includes>
                                <include>_app/IDE.html</include>
                            </includes>
                        </overlay>
                        <overlay />
                    </overlays>
                </configuration>
            </plugin>
```
But it doesn't work in this way. I guess it's because of gwt-app packaging, so GWT maven plugin assemblies war but not maven war plugin. So I'd fix it later where there will be a clear way to extend upstream packaging.
Do you know how to easily include IDE.html from upstream?

Reply:
https://github.com/redhat-developer/rh-che/blob/master/assembly/assembly-ide-war/pom.xml#L177-L197 (here it was using ide jsp file)

Reply:
It has [war](https://github.com/redhat-developer/rh-che/blob/master/assembly/assembly-ide-war/pom.xml#L23) packaging, I think It'd work even with configuring maven war plugin.

So, I tried the suggested way and it doesn't work for me, maybe I do something wrong.
https://gist.github.com/sleshchenko/d7a5377d6090ea55f47f4aec3b159a92#file-pom-xml-L333-L336
https://gist.github.com/sleshchenko/d7a5377d6090ea55f47f4aec3b159a92#file-pom-xml-L349-L370

Are you OK to merge it with copied/pasted IDE.html and fix it later?

Reply:
@benoitf https://github.com/eclipse/che/issues/7519 is merged. So I'll rework it during this PR

Reply:
ok it's better then

Reply:
@benoitf Now IDE.html is copied from assembly-ide-war